packages/cli/src/runAndroid/runAndroid.js

```diff
@@ -220,21 +212,24 @@ function startServerInNewWindow(port, terminal = process.env.REACT_TERMINAL) {
     ? `set RCT_METRO_PORT=${port}`
     : `export RCT_METRO_PORT=${port}`;
 
-  // set up the launchpackager.(command|bat) file
-  const scriptsDir = path.resolve(__dirname, '..', '..', 'scripts');
-  const launchPackagerScript = path.resolve(scriptsDir, scriptFile);
-  const procConfig: Object = { cwd: scriptsDir };
-
-  // set up the .packager.(env|bat) file to ensure the packager starts on the right port
-  const packagerEnvFile = path.join(
-    __dirname,
-    '..',
-    '..',
-    'scripts',
-    packagerEnvFilename
+  /**
+   * Set up the `.packager.(env|bat)` file to ensure the packager starts on the right port.
+   */
+  const launchPackagerScript = require.resolve(
+    `react-native/scripts/${scriptFile}`
```

Comment:
a reminder to convert this path to a flag passed to `run-android`/`run-ios` command